components/board/genericlinux/board.go

```diff
@@ -48,22 +45,18 @@ func newBoard(
 	ctx context.Context,
 	conf resource.Config,
 	gpioMappings map[int]GPIOBoardMapping,
-	usePeriphGpio bool,
 	logger golog.Logger,
 ) (board.Board, error) {
 	cancelCtx, cancelFunc := context.WithCancel(context.Background())
 	b := sysfsBoard{
-		Named:         conf.ResourceName().AsNamed(),
-		usePeriphGpio: usePeriphGpio,
-		gpioMappings:  gpioMappings,
-		logger:        logger,
-		cancelCtx:     cancelCtx,
-		cancelFunc:    cancelFunc,
-
-		spis:    map[string]*spiBus{},
-		analogs: map[string]*wrappedAnalog{},
-		// this is not yet modified during reconfiguration but maybe should be
-		pwms:       map[string]pwmSetting{},
+		Named:        conf.ResourceName().AsNamed(),
```

Comment:
is the name of this structure still good? we're not accessing the gpio pins from `/sys/class/gpio` at all, which is what the top hit on google points you to: https://www.kernel.org/doc/Documentation/gpio/sysfs.txt

Reply:
Please make a ticket to track changing this struct's name.

Reply:
I've made https://viam.atlassian.net/browse/RSDK-3951